modules/labs-react/combobox/lib/AutocompleteList.tsx

```diff
@@ -7,8 +7,10 @@ import {ComboBoxMenuItemGroup, getOptionId, listBoxIdPart, getTextFromElement} f
 import uuid from 'uuid/v4';
 
 const Autocomplete = styled('ul')({
-  padding: 0,
-  margin: `${space.xxs} 0`,
+  margin: 0,
+  maxHeight: 184,
```

Comment:
I don't think we want to change the `maxHeight` to 184 here. 200px is our standard `maxHeight` for [Menu](https://github.com/Workday/canvas-kit/blob/fa0f4d9226ada17cbd9f293eb914313d94d966d8/modules/preview-react/select/lib/SelectMenu.tsx#L160), [Combobox](https://github.com/Workday/canvas-kit/blob/3123cc7bbb0f81d3a94e6b4cd3cec3c939243564/modules/labs-react/combobox/lib/Combobox.tsx#L108), and [Dropdown](https://github.com/Workday/canvas-kit/blob/6578fc3c90f53e0afda308733858b2d3967a34bf/modules/preview-react/breadcrumbs/lib/Breadcrumbs/Dropdown/Menu.tsx#L63).

Reply:
Oh wait. I'm looking closer now, and I see the issue.

Reply:
I'm going to ask our designers what they think about compensating the height for the padding.

Reply:
```suggestion
  maxHeight: 200,
  boxSizing: 'border-box',
```

The reason you needed to adjust the `maxHeight` was because the `boxSIzing` wasn't set to `border-box`. If we do that, we can keep it at 200 and everything renders as we'd expect.

I was able to reproduce it here:
![autocomplete menu scrolling](http://g.recordit.co/VLuc0HTZLe.gif)